ansible/files/bootstrap/vault/post/810-tests.yml

```diff
@@ -3,4 +3,19 @@
 - name: "{{ service }} - post/test/ent/namespace"
   include_tasks: test/ent/namespace.yml
   when: lookup('env', '{{ service }}_enterprise') | bool
-  tags: test
\ No newline at end of file
+  tags: test
+
+- name: "Test Vault PKI endpoint OK - read pki config urls"
+  uri:
+      url: http://127.0.0.1:8200/v1/pki/config/urls
+      method: GET
+      return_content: yes
+      headers:
+          X-Vault-Token: "{{ lookup('env', 'vault_master_token') }}"
+      status_code: 200
+      body_format: json
+  when: lookup('env', 'vault_pki') | bool
+  register: pki_response
```

Comment:
Do I understand correctly that this is done because with a 200 response no output is given, but with a failure output is indeed given?

Reply:
That's right. Truth be told, it's not much to look for in this request. It's more to check if the endpoint responding.

I figured that if the `request_id` is empty something is wrong, but that's an assumption. I think we would be fine without the `until` instance.

Here's the respons:
```json
{
  "request_id": "cb5d2cb2-3b60-0fd1-cbff-3db70aa66854",
  "lease_id": "",
  "renewable": false,
  "lease_duration": 0,
  "data": {
    "crl_distribution_points": [
      "http://127.0.0.1:8200/v1/pki/crl"
    ],
    "issuing_certificates": [
      "http://127.0.0.1:8200/v1/pki/ca"
    ],
    "ocsp_servers": []
  },
  "wrap_info": null,
  "warnings": null,
  "auth": null
}
```

Reply:
I would rather like us to just remove the `until` then. The `status_code: 200` should do the trick!

Reply:
I agree - will do that!